crates/astria-conductor/src/executor.rs

```diff
@@ -231,21 +244,34 @@ impl<C: ExecutionClientV1Alpha1> Executor<C> {
 
         let response = self
             .execution_rpc_client
-            .call_do_block(prev_block_hash, block.rollup_transactions, Some(timestamp))
+            .call_execute_block(prev_block_hash, block.rollup_transactions, Some(timestamp))
             .await?;
-        self.execution_state = response.block_hash.clone();
 
         // store block hash returned by execution client, as we need it to finalize the block later
         info!(
             sequencer_block_hash = ?block.block_hash,
             sequencer_block_height = block.header.height.value(),
-            execution_block_hash = hex::encode(&response.block_hash),
+            execution_block_hash = hex::encode(&response.hash),
             "executed sequencer block",
         );
         self.sequencer_hash_to_execution_hash
-            .insert(block.block_hash, response.block_hash.clone());
+            .insert(block.block_hash, response.hash.clone());
 
-        Ok(Some(response.block_hash))
+        Ok(Some(response.hash))
```

Comment:
I feel like we should update `execute_block` to return the response (a block) here. Right now we are having to rebuild the block instead of just using the return type when updating the state commitment.